client/app/views/main.coffee

```diff
@@ -128,6 +128,36 @@ module.exports = class HomeView extends BaseView
         @displayView @configApplications
         window.document.title = t "cozy applications title"
 
+    displayUpdateApplication: (slug) =>
+        @displayView @configApplications
+        window.document.title = t "cozy applications title"
+        window.app.routers.main.navigate 'config-applications', false
+
+        # When the route is called on browser loading, it must wait for
+        # apps list to be retrieved
+        if @apps.length is 0
```

Comment:
Sorry that sounds logic that if you have no app, you can't update it. You could add a warning alert/modal but that's not really important.